app/webpacker/components/CompetitionsOverview/CompetitionsFilters.js

```diff
@@ -159,7 +159,7 @@ function TimeOrderButtonGroup({ filterState, dispatchFilter }) {
   return (
     <>
       <label htmlFor="state">{I18n.t('competitions.index.state')}</label>
-      <Button.Group id="state" size="small" compact primary>
+      <Button.Group id="state" size="large" primary>
```

Comment:
Why? This change can potentially affect other pages as well...

Reply:
you did this change!

Reply:
There's a chance >50% that was an accident.